draft-packages/tag/docs/Tag.stories.tsx

```diff
@@ -31,117 +23,136 @@ export default {
+        <Tag variant="statusLive" size="medium">
+          Live
+        </Tag>
+        <Tag variant="statusDraft" size="medium">
+          Draft
+        </Tag>
+      </StoryWrapper.Row>
+      <StoryWrapper.Row gridColumns={10} rowTitle="Status (sm)">
+        <Tag variant="default" size="small">
+          Report
+        </Tag>
+        <Tag variant="statusLive" size="small">
+          Live
+        </Tag>
+        <Tag variant="statusDraft" size="small">
+          Draft
+        </Tag>
+      </StoryWrapper.Row>
+    </StoryWrapper>
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.RowHeader
+        gridColumns={10}
+        headings={["Neutral", "Postive", "Negative"]}
+      />
+      <StoryWrapper.Row gridColumns={10} rowTitle="Sentiment (md)">
+        <Tag variant="sentimentNeutral" size="medium">
+          Report
+        </Tag>
+        <Tag variant="sentimentPositive" size="medium">
+          Live
+        </Tag>
+        <Tag variant="sentimentNegative" size="medium">
+          Draft
+        </Tag>
+      </StoryWrapper.Row>
+      <StoryWrapper.Row gridColumns={10} rowTitle="Sentiment (sm)">
+        <Tag variant="sentimentNeutral" size="small">
+          Report
+        </Tag>
+        <Tag variant="sentimentPositive" size="small">
+          Live
+        </Tag>
+        <Tag variant="sentimentNegative" size="small">
+          Draft
+        </Tag>
+      </StoryWrapper.Row>
+    </StoryWrapper>
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.RowHeader
+        headings={["Informative", "Postive", "Negative", "Cautionary"]}
+        gridColumns={7}
+      />
+      <StoryWrapper.Row gridColumns={7} rowTitle="Validation (md)">
+        <Tag variant="validationInformative" size="medium">
+          Informative
+        </Tag>
+        <Tag variant="validationPositive" size="medium">
+          Positive
+        </Tag>
+        <Tag variant="validationNegative" size="medium">
+          Negative
+        </Tag>
+        <Tag variant="validationCautionary" size="medium">
+          Cautionary
+        </Tag>
+      </StoryWrapper.Row>
+    </StoryWrapper>
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.RowHeader gridColumns={10} headings={["Base"]} />
+      <StoryWrapper.Row gridColumns={10} rowTitle="Profile (md)">
+        <Tag
+          variant="profile"
+          avatar={
+            <Avatar
+              size="small"
+              avatarSrc="https://www.cultureampcom-preview-1.usw2.wp-dev-us.cultureamp-cdn.com/assets/slices/main/assets/public/media/chapters-card-1@2x.05e547444387f29f14df0b82634bf2b6.png"
+              fullName="Jane Doe"
+            />
+          }
+          dismissible
+        >
+          Jane Doe
+        </Tag>
+      </StoryWrapper.Row>
+    </StoryWrapper>
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.Row gridColumns={10} rowTitle="Dismissible (md)">
+        <Tag variant="default" size="medium" dismissible>
```

Comment:
```suggestion
      <StoryWrapper.Row gridColumns={10} rowTitle="Dismissible (sm)">
        <Tag variant="default" size="small" dismissible>
```

UI Kit appears to be like this